src/components/drawer/drawer.stories.mdx

```diff
@@ -3,6 +3,13 @@ import Drawer from '.';
 import { useState, useCallback } from 'react';
 import { text } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
+import FlatTable from '../flat-table/flat-table.component';
+import FlatTableHead from '../flat-table/flat-table-head/flat-table-head.component';
+import FlatTableBody from '../flat-table/flat-table-body/flat-table-body.component';
+import FlatTableRow from '../flat-table/flat-table-row/flat-table-row.component';
+import FlatTableHeader from '../flat-table/flat-table-header/flat-table-header.component';
+import FlatTableCell from '../flat-table/flat-table-cell/flat-table-cell.component';
+import FlatTableRowHeader from '../flat-table/flat-table-row-header/flat-table-row-header.component';
```

Comment:
I think it can be just 
```js
import {
  FlatTable, 
  FlatTableHead,
  FlatTableBody,
  FlatTableRow,  
  FlatTableHeader, 
  FlatTableCell,
  FlatTableRowHeader
} from '../flat-table'
```

Reply:
indeed, updated